### PR TITLE
Add new threat detection documentation for AppG

### PR DESCRIPTION
**Pull request by sranjit9**

### CODE REVIEW

**Comment by PRMerger19** [comment]
@sranjit9 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@sranjit9 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jborsecnik** [comment]
@dcurwin, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. Thanks.

**Comment by PRMerger3** [comment]
@sranjit9 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/app-governance-anomaly-detection-alerts.md [hunk 1]
```diff
@@ -578,6 +578,30 @@ This detection identifies that an App consented to high privilege scope, created
...
 
+### App made high volume of OneDrive search and edit activities 
+
+**Severity**: Medium
+
+**MITRE ID’s**: T1137, T1213
+
+This detection identifies an OAuth app with high privilege permissions performing high volume of searches and edits to OneDrive files using Graph API.
+
+**TP or FP?**
```
[the remark this answers is not in the document]
Comment (sranjit9) [reply CloudAppSecurityDocs/app-governance-anomaly-detection-alerts.md:589]: I have included the text you have suggested above, but I kept the structure same to maintain the format as other detections in the document. But, I agree, the document could be restructured.

**Review by dcurwin — APPROVED** [review]

**Comment by Court72** [comment]
@dcurwin

Thanks for the review! Can you add a `#sign-off` comment to signal that this PR can be merged?

**Comment by dcurwin** [comment]
@sranjit9 should sign off when this is ready to publish.

**Comment by PRMerger12** [comment]
@sranjit9: I'm sorry - only the author of this article, @dcurwin, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger18** [comment]
@jborsecnik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/app-governance-anomaly-detection-alerts.md [hunk 2]
```diff
@@ -578,6 +578,31 @@ This detection identifies that an App consented to high privilege scope, created
...
+
+**Severity**: Medium
+
+**MITRE IDs**: T1137, T1213
+
+This detection identifies OAuth apps with high privilege permissions that perform a large number of searches and edits in OneDrive using Graph API.
+
+**TP or FP?**
+
+- **TP**: If you’re able to confirm that a high usage of OneDrive workload via Graph API is not expected from this OAuth application having high privilege permissions to read and write to OneDrive, then a true positive is indicated.
```
Comment (lomayor) [inline CloudAppSecurityDocs/app-governance-anomaly-detection-alerts.md:591]: If you’re able to confirm that the increased activity on OneDrive is not expected from this privileged OAuth app, we recommend handling this alert as a true positive.

File: CloudAppSecurityDocs/app-governance-anomaly-detection-alerts.md [hunk 3]
```diff
@@ -578,6 +578,31 @@ This detection identifies that an App consented to high privilege scope, created
...
+
+This detection identifies OAuth apps with high privilege permissions that perform a large number of searches and edits in OneDrive using Graph API.
+
+**TP or FP?**
+
+- **TP**: If you’re able to confirm that a high usage of OneDrive workload via Graph API is not expected from this OAuth application having high privilege permissions to read and write to OneDrive, then a true positive is indicated. 
+
+  **Recommended Action**: Based on the investigation, if the application is malicious, you can revoke consents and disable the application in the tenant. If it is a compromised application, you can revoke the consents, temporarily disable the app, review the required permissions, reset the password and then re-enable the app.
+
+- **FP**: If after investigation, you can confirm that the app has a legitimate business use in the organization.
```
Comment (lomayor) [inline CloudAppSecurityDocs/app-governance-anomaly-detection-alerts.md:595]: If your investigation indicates that the increased activity on OneDrive from this privileged OAuth app is legitimate, handle this alert as a false positive.

**Pull request merged**